Name: Hide Airplane Mode Icon
Version: 2.0.1-1
Author: Jason Robitaille
Description: Hides the airplane mode icon from the top bar.  (MIT license)

--- .orig/usr/lib/luna/system/luna-systemui/stylesheets/systemui.css
+++ /usr/lib/luna/system/luna-systemui/stylesheets/systemui.css
@@ -184,7 +184,7 @@
 #rssi.bars-3 { background-image: url(../images/rssi-3.png); }
 #rssi.bars-4 { background-image: url(../images/rssi-4.png); }
 #rssi.bars-5 { background-image: url(../images/rssi-5.png); }
-#rssi.flight-mode { background-image: url(../images/rssi-flightmode.png); }
+#rssi.flight-mode { background-image: url(../images/rssi-flightmode.png); display:none; }
 #rssi.error { background-image: url(../images/rssi-error.png); }
 
 #roaming {
